index.js

```diff
@@ -23,17 +24,13 @@ const processor = (report, reporterOptions = {}, jestRootDir = null) => {
 
   const jsonResults = buildJsonResults(report, fs.realpathSync(process.cwd()), options);
 
-  // Set output to use new outputDirectory and fallback on original output
```

Comment:
This block is refactored into `./utils.getOutputPath.js` for cleaniness. I can move it back if you prefer it that way.